File: messenger-client/Models/LoginResponse.cs
using Messenger.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Messenger.Models
{
    public class LoginResponse
    {
        public Boolean successful { get; set; }
        public String message { get; set; }
        public User user { get; set; }
    }
}

File: messenger-api/messenger/src/main/java/com/chat/messenger/model/response/LoginResponse.java
package com.chat.messenger.model.response;

import com.chat.messenger.model.dto.UserDto;
import lombok.Data;

@Data
public class LoginResponse {

    private boolean successful;
    private String message;
    private UserDto user;

    public LoginResponse(boolean successful, String message, UserDto user) {
        this.successful = successful;
        this.message = message;
        this.user = user;
    }
}
